mlflow/gateway/providers/bedrock.py

```diff
@@ -243,7 +253,7 @@ def _underlying_provider(self):
         return AmazonBedrockModelProvider.of_str(provider)
 
     @property
-    def underlying_provider_adapter(self) -> ProviderAdapter:
+    def adapter(self) -> ProviderAdapter:
```

Comment:
Does this method return an instance of `ProviderAdapter`?

Reply:
Nope, will update to `Type[ProviderAdapter]`